.sass-lint.json

```diff
@@ -0,0 +1,78 @@
+{
+  "options": {
+    "formatter": "stylish",
+    "merge-default-rules": false
+  },
+  "files": {
+    "include": "+(addon|app|tests)/styles/**/*.s+(a|c)ss"
+  },
+  "rules": {
+    "extends-before-mixins": 2,
+    "extends-before-declarations": 2,
+    "placeholder-in-extend": 2,
+    "mixins-before-declarations": 2,
+    "one-declaration-per-line": 2,
+    "empty-line-between-blocks": 2,
+    "single-line-per-selector": 2,
+    "no-color-keywords": 2,
+    "no-color-literals": 2,
+    "no-css-comments": 2,
+    "no-debug": 2,
+    "no-duplicate-properties": 2,
+    "no-empty-rulesets": 2,
+    "no-extends": 0,
+    "no-ids": 2,
+    "no-important": 2,
+    "no-invalid-hex": 2,
+    "no-mergeable-selectors": 2,
+    "no-misspelled-properties": 2,
+    "no-qualifying-elements": 2,
+    "no-trailing-whitespace": 2,
+    "no-trailing-zero": 2,
+    "no-transition-all": 2,
+    "no-url-protocols": 2,
+    "no-vendor-prefixes": 2,
+    "no-warn": 2,
+    "property-units": 0,
+    "force-attribute-nesting": 2,
+    "force-element-nesting": 2,
+    "force-pseudo-nesting": 2,
+    "class-name-format": 2,
+    "function-name-format": 2,
+    "id-name-format": 0,
+    "mixin-name-format": 2,
+    "placeholder-name-format": 2,
+    "variable-name-format": 2,
+    "bem-depth": 0,
+    "border-zero": 2,
+    "brace-style": 2,
+    "clean-import-paths": 2,
+    "empty-args": 2,
+    "hex-length": 2,
+    "hex-notation": 2,
+    "indentation": 2,
+    "leading-zero": 2,
+    "nesting-depth": 0,
```

Comment:
I thought by default we wanted a nesting depth defined. We've had to override in a few apps to get them to work until they can be refactored, but we probably shouldn't default to not checking it.